config/default.schema.json

```diff
@@ -1059,6 +1065,71 @@
 				}
 			}
 		},
+		"SESSION": {
+			"type": "object",
+			"description": "Cookie properties, required always to be defined",
+			"properties": {
+				"SAME_SITE": {
+					"type": "string",
+					"default": "lax",
+					"enum": ["none", "lax", "strict"],
+					"description": "Value for session cookies sameSite property. When SECURE flag is false, 'None' is not allowed in SAME_SITE and Lax should be used as default instead"
+				},
+				"SECURE": {
+					"type": "boolean",
+					"default": true,
+					"description": "Value for session cookies httpOnly property"
+				},
+				"HTTP_ONLY": {
+					"type": "boolean",
+					"default": true,
+					"description": "Value for session cookies httpOnly property"
+				},
+				"EXPIRES_SECONDS": {
```

Comment:
how does this value interfere with the expiration set for the jwt timeout?

Reply:
Not at all. This is the expiration of the session cookie and has no connection to the jwt.